package.json

```diff
@@ -37,17 +37,13 @@
     "@curveball/bodyparser": "^0.6.1",
     "@curveball/core": "^0.21.1",
     "@curveball/http-errors": "^0.5.0",
-    "@types/chai": "^4.3.1",
-    "@types/mocha": "^10.0.0",
     "@types/node": "^18.0.0",
     "@typescript-eslint/eslint-plugin": "^6.2.0",
     "@typescript-eslint/parser": "^6.2.0",
-    "chai": "^4.3.6",
     "eslint": "^8.1.0",
-    "mocha": "^10.0.0",
     "node-fetch": "^2.6.7",
     "ts-loader": "^9.2.6",
-    "ts-node": "^10.7.0",
+    "tsx": "^3.14.0",
```

Comment:
Using v3 b/c node-16 support dropped in semver [v4](https://github.com/privatenumber/tsx/releases/tag/v4.0.0).